README.md

```diff
@@ -1,20 +1,25 @@
-# Rust on Android
+> ⚠️ **Can't find what you're looking for?** ⚠️
+>
+> This repository was recently [modularized](https://github.com/rust-mobile/ndk/issues/372) and the following crates were split into separate repositories:
+>
+> Crate | New Location | Notes
+> ------|--------------|------
+> ndk-context | https://github.com/rust-mobile/ndk-context |
+> ndk-glue | https://github.com/rust-mobile/ndk-glue | _deprecated_ - see [android-activity](https://github.com/rust-mobile/android-activity)
+> ndk-build | https://github.com/rust-mobile/cargo-apk |
+> cargo-apk | https://github.com/rust-mobile/cargo-apk |
```

Comment:
Don't think anyone uses `ndk-macro` but we could mention it... Otherwise drop the pointer to `ndk-build` as well?

Reply:
ah yep, forgot about ndk-macros - I think I just went by the list of crates that were removed from the previous table in the README. I'll add a line for ndk-macros.